tools/java/org/unicode/cldr/util/XPathParts.java

```diff
@@ -738,8 +776,7 @@ public Element(String element, Map<String, String> attributes) {
 
         @Override
         protected Object clone() throws CloneNotSupportedException {
-            return frozen ? this
-                : new Element(element, attributes);
+            return new Element(element, attributes);
```

Comment:
I don't think we actually need to call clone() anymore; if so this method can be dropped.

Reply:
Best is to inline; that will catch places where used, and we can then see if they can be optimized.